packages/client/src/intros/shared/Intro.tsx

```diff
@@ -76,25 +76,24 @@ export default ({
       {exceptions.length > 0 && (
         <>
           <Heading styleAs="h4" forwardedAs="h3">
-            {t("introPage.common.exceptions title")}
+            <em>{t("introPage.common.exceptions title")}</em>
           </Heading>
-          <Paragraph gutterBottom={8}>
-            {t("introPage.common.exceptions description")}
-          </Paragraph>
+          {showContactInformation && (
+            <Paragraph>
+              <ContactSentence eventName={sections.INTRO} />
```

Comment:
There are two `<ContactSentence />` components in this page. I think the other one is obsolete

Reply:
That other one if for the case there are not exceptions.

Reply:
The other one is obsolete